app/src/main/java/com/gh4a/adapter/timeline/EventViewHolder.java

```diff
@@ -249,6 +241,29 @@ protected Intent getIntent() {
             text.setSpan(new IssueLabelSpan(mContext, label, false), pos, pos + length, 0);
         }
 
+        CharSequence time = event.getCreatedAt() != null
+                ? StringUtils.formatRelativeTime(mContext, event.getCreatedAt(), true) : "";
+
+        pos = text.toString().indexOf("[time]");
+        if (pos >= 0) {
+            text.replace(pos, pos + 6, time);
+            if (event.getCreatedAt() != null) {
+                text.setSpan(new ClickableSpan() {
+                    @Override
+                    public void updateDrawState(TextPaint ds) {
+                        // No special stying
+                    }
+
+                    @Override
+                    public void onClick(View widget) {
+                        CharSequence longTime = StringUtils.formatExactTime(mContext,
+                                event.getCreatedAt());
+                        Toast.makeText(mContext, longTime, Toast.LENGTH_LONG).show();
```

Comment:
Outside the scope of this PR: this may be worth adding to separate timestamp fields as well, isn't it?

Reply:
I don't understand what do you mean here.

Reply:
I mean adding the functionality of showing a toast with the exact timestamp also to timestamps that have their own TextView, like e.g. for issue comments.